### ISSUE TITLE
System tray icon blocks taskbar hiding on every song change

### ISSUE THREAD

**Issue by TouchMonkey** [adammhaile/Elpis#98]
Elpis breaks auto-hiding of the Window Explorer taskbar. Every time the song changes, the next time the taskbar is displayed it will not auto-hide until you click/right-click the Elpis system tray icon to acknowledge the notification.

Repro Steps:
1. Enable taskbar hiding (Right-click the taskbar, select Properties, check "Auto-hide the taskbar" in the Taskbar tab)
2. Put the taskbar in a hidden state by moving the mouse away from the bottom of the screen.
3. Wait for Elpis to change songs
4. Move the mouse to the bottom of the screen, which displays the taskbar, then away again.

Expected Result:
The taskbar should hide itself after moving the mouse away from the bottom edge of the screen

Actual Result:
The taskbar will not hide, and the Elpis icon remains displayed in the System Tray until you acknowledge the notification by right-clicking the icon then clicking away again (or left-clicking but that pauses the song which isn't usually desired).

My educated guess (could be wrong) is that the song information that is displayed in the system tray icon's context menu is getting updated every time the song changes (or the hover-over text), and Explorer treats the update as something that requires acknowledgement. If that's the case, the solution would be to change the menu/hover text on demand instead of doing it automatically.

**Comment by seliver** [comment]
Good point. I think I did experienced that, but never really considered a problem. Will see whats going on with that.

**Comment by seliver** [comment]
Looks like this is not an issue on windows 10 - https://stackoverflow.com/questions/8241479/c-sharp-nofityicon-balloon-tip-doesnt-always-go-away-within-specified-time#. 
Can someone check if this still happens on windows 10? Here, windows 8.1, still happens.

**Comment by justoned420** [comment]
Also happening on Win7 x64

**Comment by Wires77** [comment]
I don't have access to a Windows 7 machine to test this, but if one of you would like to submit a PR, I can accept that
